website/projects/tests/tests_aws/test_awsapitalker.py

```diff
@@ -0,0 +1,198 @@
+                "EvaluationResults"
+            ]
+
+        self.assertEquals(eval_results[0]["EvalDecision"], "allowed")
+
+    def test_describe_organization(self):
+        self.create_organization()
+
+        response = self.api_talker.describe_organization()
+
+        self.assertIn("Organization", response)
+        self.assertIn("MasterAccountId", response["Organization"])
+        self.assertIn("MasterAccountEmail", response["Organization"])
+
+    def test_describe_policy(self):
+        self.create_organization()
+
+        policy_id = self.create_dummy_policy()
+
+        policy = self.api_talker.describe_policy(policy_id)["Policy"]
+        policy_summary = policy["PolicySummary"]
+        policy_content = self.create_dummy_policy_content()
+
+        self.assertEquals(policy_summary["Name"], "Test policy")
+        self.assertEquals(policy_summary["Description"], "Policy for testing purposes")
+        self.assertEquals(policy_content, policy["Content"])
+
+    def test_create_account(self):
+        moto_client = boto3.client("organizations")
+
+        self.create_organization()
+
+        response = self.api_talker.create_account("test@example.com", "Test")
+
+        accounts = moto_client.list_accounts()["Accounts"]
+
+        self.assertEquals(response["CreateAccountStatus"]["AccountName"], "Test")
+        self.assertIn(("Test", "test@example.com"), [(account["Name"], account["Email"]) for account in accounts])
+
+    def test_move_account(self):
+        moto_client = boto3.client("organizations")
+
+        org_id = self.create_organization()
+
+        account_status = self.api_talker.create_account("test@example.com", "Test")
+        account_id = account_status["CreateAccountStatus"]["AccountId"]
+
+        source_ou_info = self.api_talker.create_organizational_unit(org_id, "Source OU")
+        source_ou_id = source_ou_info["OrganizationalUnit"]["Id"]
+        dest_ou_info = self.api_talker.create_organizational_unit(org_id, "Destination OU")
+        dest_ou_id = dest_ou_info["OrganizationalUnit"]["Id"]
+
+        self.api_talker.move_account(account_id, source_ou_id, dest_ou_id)
+
+        accounts_under_source = moto_client.list_children(ParentId=source_ou_id, ChildType="ACCOUNT")["Children"]
+        accounts_under_dest = moto_client.list_children(ParentId=dest_ou_id, ChildType="ACCOUNT")["Children"]
+        self.assertNotIn(account_id, [account["Id"] for account in accounts_under_source])
+        self.assertIn(account_id, [account["Id"] for account in accounts_under_dest])
+
+    def test_list_organizational_units_for_parent(self):
+        self.create_organization()
+
+        root_id = self.api_talker.list_roots()[0]["Id"]
+
+        ou_1 = self.api_talker.create_organizational_unit(root_id, "Test OU 1")
+        ou_2 = self.api_talker.create_organizational_unit(root_id, "Test OU 2")
+
+        received_ous = self.api_talker.list_organizational_units_for_parent(root_id)
```

Comment:
Maybe change to received_ou_list, expected_ou_list, ous is not readable

Reply:
Fixed.